armci/src/common/capi.c

```diff
@@ -208,15 +208,6 @@ int ARMCI_Init_mpi_comm(MPI_Comm comm)
 #endif
 
 
-#if HAVE_SYS_WEAK_ALIAS_PRAGMA
-#   pragma weak ARMCI_Init
```

Comment:
at the very least, we have a problem if the weak symbol doesn't match the name of the function, no?